sdk/python/packages/flet-core/src/flet_core/cupertino_segmented_button.py

```diff
@@ -125,8 +125,8 @@ def controls(self) -> List[Control]:
         return self.__controls
 
     @controls.setter
-    def controls(self, value: List[Control]):
-        self.__controls = value
+    def controls(self, value: Optional[Sequence[Control]]):
```

Comment:
Please remove the optional here.
In the line below just do `self.__controls = list(value)` (no check for `None`, as this is a required value)

Reply:
The relevant content has been modified accordingly for these four items.

![image](https://github.com/user-attachments/assets/7e25e9a5-f7d1-4861-8239-e3eb472d0064)